features/step_definitions/animal_steps.rb

```diff
@@ -1 +1,35 @@
-#Delete this comment, here is where you should write your step defs
+Given(/^Trudy/) do
+  @animal = Animal.new('Trudy', 'lemur', 17)
+end
+
+When(/^we track (?:him|her)/) do
+  @age = @animal.age
+  @name = @animal.name
+  @type = @animal.type
+  @old = @animal.old?
+end
+
+Then(/^the age should be (\d+)$/) do |age|
+  expect(@age).to eq(age.to_i)
+end
+
+And(/^the name should be (\w+)/) do |name|
+  expect(@name).to eq(name)
+end
+
+And(/^the type should be (\w+)/) do |type|
+  expect(@type).to eq(type)
+end
+
+And(/^she should be old/) do
+  expect(@old).to be true
+end
+
+
+Given(/^Oscar/) do
```

Comment:
Is it possible to dynamically create this animal based on the rest of the string?